one-ui/src/main/java/com/marklogic/hub/oneui/controllers/EnvironmentController.java

```diff
@@ -112,8 +115,13 @@ public void onError(String commandName, Exception exception) {
         String directory = payload.get("directory").asText("");
         // setting the project directory will resolve any relative paths
         try {
+            Path directoryPath = Paths.get(directory);
             if (StringUtils.isEmpty(directory)) {
                 throw new BadRequestException("Missing required directory property");
```

Comment:
Same thing - I think it's good to qualify this as the "Project directory" - e.g.

    "Project directory not specified"